hack/fetch-releases.sh

```diff
@@ -20,7 +20,11 @@ release_yaml() {
      dir="trigger"
     fi
 
-    #nightly -> 0.0.0-nightly
+    if [[ ${releaseFileName} == "tekton-dashboard-release-readonly" ]]; then
+      dir="dashboard-readonly"
```

Comment:
could we create sub-directories in `dashboard` dir instead of creating a new directory. 
I would prefer that becuase operator-dev has an informal agreement that we keep 1 top level directory per TektonComponent. 🧑‍💻